crates/hyperswitch_connectors/src/connectors/novalnet/transformers.rs

```diff
@@ -378,6 +555,24 @@ pub fn get_error_response(result: ResultData, status_code: u16) -> ErrorResponse
     }
 }
 
+impl NovalnetPaymentsResponseTransactionData {
+    pub fn get_token(transaction_data: Option<&Self>) -> Option<String> {
+        if let Some(data) = transaction_data {
+            match &data.payment_data {
+                Some(NovalnetResponsePaymentData::Card(card_data)) => {
+                    card_data.token.clone().map(|token| token.expose())
+                }
+                Some(NovalnetResponsePaymentData::Paypal(paypal_data)) => {
```

Comment:
will we get card_data incase of gpay & apple pay mandate as well ?

Reply:
yes